src/sentry/integrations/slack/message_builder/organization_requests.py

```diff
@@ -1,30 +0,0 @@
-from typing import TYPE_CHECKING, Any, Mapping, Union
-
-from sentry.integrations.slack.message_builder import SlackBody
-from sentry.integrations.slack.message_builder.notifications import SlackNotificationsMessageBuilder
-from sentry.notifications.notifications.organization_request import OrganizationRequestNotification
-
-if TYPE_CHECKING:
-    from sentry.models import Team, User
-
-
-class SlackOrganizationRequestMessageBuilder(SlackNotificationsMessageBuilder):
-    def __init__(
```

Comment:
Hmm this is gonna need a rebase when Steve's PR is merged since he's adding to this file